keyboards/madjax_macropad/keyboard.json

```diff
@@ -0,0 +1,55 @@
+{
+    "manufacturer": "MadJax",
+    "keyboard_name": "madjax_macropad",
+    "maintainer": "guidoism",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
```

Comment:
```suggestion
        "nkro": true
```

Reply:
Changed back to strict json as requested.